src/posthog-core.js

```diff
@@ -1488,6 +1523,9 @@ PostHogLib.prototype['decodeLZ64'] = PostHogLib.prototype.decodeLZ64
 PostHogLib.prototype['SentryIntegration'] = PostHogLib.prototype.sentry_integration
 PostHogLib.prototype['debug'] = PostHogLib.prototype.debug
 PostHogLib.prototype['LIB_VERSION'] = Config.LIB_VERSION
+PostHogLib.prototype['startSessionRecording'] = PostHogLib.prototype.startSessionRecording
```

Comment:
Q: Why do we want to expose these at the top level in the first place? Vs using it via `posthog.sessionRecording` - it keeps the core more lean.